screen-orientation/hidden_document.html

```diff
@@ -14,48 +14,22 @@
 
   promise_test(async (t) => {
     const { minimize, restore } = window_state_context(t);
+    t.add_cleanup(restore);
 
     await minimize();
 
     assert_equals(document.visibilityState, "hidden", "Document must be hidden");
-    await promise_rejects_dom(t, "SecurityError", screen.orientation.lock("landscape") );
+    const opposite = getOppositeOrientation();
+    await promise_rejects_dom(t, "SecurityError", screen.orientation.lock(opposite) );
   }, "hidden documents must reject went trying to call lock or unlock");
 
   promise_test(async (t) => {
     const { minimize, restore } = window_state_context(t);
+    t.add_cleanup(restore);
```

Comment:
Please remove this since I remove this by https://github.com/web-platform-tests/wpt/pull/38031

Reply:
Oops, done!